### PR TITLE
add typings and stuff

### PR DESCRIPTION
**Pull request by Uzlopak**

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+declare namespace fastifyChangeMe {
+  export interface FastifyChangeMeOptions {}
+
+  export const fastifyChangeMe: FastifyChangeMe
+  export { fastifyChangeMe as default }
+}
+
+declare function fastifyChangeMe(...params: Parameters<FastifyChangeMe>): ReturnType<FastifyChangeMe>
+
+export = fastifyChangeMe
```
Comment (Uzlopak) [inline types/index.d.ts:14]:
```suggestion
export = fastifyChangeMe

```

File: package.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
   "scripts": {
```
Comment (Eomm) [inline package.json:6]: We need to add types and rename `name` to let it works
```suggestion
  "name": "change-me",
  "types": "types/index.d.ts",
  "scripts": {
```

Comment (Uzlopak) [reply package.json:6]: we had it already above. so i patched it

Comment (Eomm) [reply package.json:6]: I got this:

```
The type definition `index.d.ts` does not exist at `/Users/mspigolon/workspace/_experiments/fastify-plugin-template/index.d.ts`. Is the path correct? Create one and try again.
```

Comment (Uzlopak) [reply package.json:7]: fixed

File: package.json [hunk 3]
```diff
@@ -1,12 +1,15 @@
 {
-  "name": "<CHANGE ME>",
+  "name": "change-me",
   "version": "0.0.1",
   "description": "<CHANGE ME>",
   "main": "index.js",
+  "types": "types/indey.d.ts",
```
Comment (Eomm) [inline package.json:6]:
```suggestion
  "types": "types/index.d.ts",
```

**Review by Eomm — APPROVED** [review]

**Pull request merged**